guides/common/modules/proc_configuring-project-for-puppet-compliance-policy-deployment.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-{project-context}-for-puppet-compliance-policy-deployment"]
+= Configuring {Project} for Puppet compliance policy deployment
+
+If you want to use Puppet to deploy compliance policies, configure {Project} for Puppet compliance policy deployment.
+
+.Procedure
+. Ensure Puppet is enabled.
```

Comment:
This is a prerequisite.

Reply:
https://github.com/theforeman/foreman-documentation/issues/4214